dictdiffer/_compat.py

```diff
@@ -5,27 +5,9 @@
 # Dictdiffer is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more
 # details.
+from collections import Iterable, MutableMapping, MutableSequence, MutableSet
+from itertools import zip_longest as izip_longest
 
-"""Python compatibility definitions."""
-
-try:
-    PY2 = True
-    string_types = basestring,
-    text_type = unicode
-    num_types = int, long, float
-except NameError:
-    PY2 = False
-    string_types = str,
-    text_type = str
-    num_types = int, float
-
-if PY2:
-    from collections import (Iterable, MutableMapping, MutableSequence,
-                             MutableSet)
-    from itertools import izip_longest as _zip_longest
-else:
-    from collections import (Iterable, MutableMapping, MutableSequence,
-                             MutableSet)
-    from itertools import zip_longest as _zip_longest
-
-izip_longest = _zip_longest
+string_types = str,
+text_type = str
+num_types = int, float
```

Comment:
We should probably completely remove this file. It would be nice to do this change in a separate PR which would remove the support for PY<3.5.

Reply:
Yeah, as you prefer.
Do you want me to revert this file?

Reply:
Revert here or remove in separate PR as you prefer.

Reply:
Ok, I am going to do another PR to drop support of Py<3.5 then I will rebase this one on it.

Reply:
See #160